lib/views/home_screen.dart

```diff
@@ -178,7 +178,7 @@ class _MainScreenState extends State<MainScreen> with TickerProviderStateMixin {
                                 width: 45.w,
                                 child: HikeButton(
                                   buttonWidth: homebwidth,
-                                  buttonHeight: homebheight - 2,
+                                  buttonHeight: homebheight - 13.5,
```

Comment:
this change won't fix the issue. The issue is wrt text length and button width and height. It works fine on some devices with more width and in some cases the text goes into next line.
pls think of some way to make text more adaptive which can work on all devices